Makefile

```diff
@@ -305,5 +305,10 @@ env: ## Generate env files
 	@echo "$(P) Generate/Regenerate env files required for application (generated passwords only match if database .env file does not already exist)"
 	@./scripts/gen-env-files.sh;
 
-.PHONY: logs start destroy local setup restart refresh up down stop build rebuild clean client-test server-test pause-30 server-run db-clean db-drop db-seed db-refresh db-script db-scaffold npm-clean npm-refresh keycloak-sync convert backend-coverage frontend-coverage backend-test frontend-test env
+mayan-up: ## Calls the docker compose up for the mayan images
```

Comment:
not 100% sure this would be useful, but add it for to match the rest of the dev tools